pkg/controller/status/controller.go

```diff
@@ -32,14 +32,21 @@ const (
 	// as InsightsUploadDegraded
 	uploadFailuresCountThreshold = 5
 
-	AsExpectedReason         = "AsExpected"
-	degradedReason           = "Degraded"
-	noTokenReason            = "NoToken"
-	upgradeableReason        = "InsightsUpgradeable"
-	insightsAvailableMessage = "Insights works as expected"
-	reportingDisabledMsg     = "Health reporting is disabled"
-	monitoringMsg            = "Monitoring the cluster"
-	canBeUpgradedMsg         = "Insights operator can be upgraded"
+	// Condition reasons
+	AsExpectedReason      = "AsExpected"
+	degradedReason        = "Degraded"
+	noTokenReason         = "NoToken"
+	disabledWithAPIReason = "DisabledWithApi"
+	upgradeableReason     = "InsightsUpgradeable"
+
+	// Condition messages
+	insightsAvailableMessage  = "Insights works as expected"
+	reportingDisabledMessage  = "Health reporting is disabled"
+	monitoringMessage         = "Monitoring the cluster"
+	canBeUpgradedMessage      = "Insights operator can be upgraded"
+	disabledWithAPIMessage    = "Gathering is disabled via API"
```

Comment:
Perhaps here we can be more specific? Disabled in `insightsdatagather.config.openshift.io` or something similar? Just a suggestion.

Reply:
Changed to `Gathering is disabled in insightsdatagather.config.openshift.io`